.github/workflows/infrautils-cfn-deploy.yml

```diff
@@ -0,0 +1,70 @@
+# This workflow installs infrautils (DevOps maintained infrastructure as service python package) from JFrog
+# and uses infrautils (boto3) to deploy a cloudformation stack update
+# https://github.com/energyhub/infra-utils
+name: Infrautils CFN Deploy
+
+on:
+  workflow_call:
+    secrets:
+      JFROG_CI_PASSWORD:
```

Comment:
Should we be using `JFROG_CI_PASSWORD` instead of `JFROG_PASSWORD`, which is used in all the repos now?

Reply:
I am not 100% certain but the JFROG_CI_PASSWORD is for the ci-infra-utils user which has access to the pypi repos - I believe the JFROG_PASSWORD that's in all the repos now is for maven rather than pypi? I will double check.

Reply:
Oh ya I think you're right. well `JFROG_PASSWORD` is so many places I guess. maybe this one could be `JFROG_PYPY_PASSWORD`? not sure what the clearest way to do that is.